release/create_release_archives.sh

```diff
@@ -131,10 +131,15 @@ for unwanted_manifest in \
     istio-one-namespace-auth.yaml \
     istio-multicluster.yaml \
     istio-auth-multicluster.yaml \
+    istio.yaml \
+    addons/zipkin.yaml \
```

Comment:
See @douglas-reid 's PR for Jaeger.. I guess instead of Zipkin, we will have Jaeger..